docs/LIMITS.md

```diff
@@ -55,8 +55,8 @@ supported:
   S3 Find and Forget deployment
 - S3 Objects encrypted with SSE-C are not supported
 - If the bucket targeted by a data mapper belongs to an account other than the
-  account that the Amazon S3 Find and Forget Solution is deployed in, a CMK must
-  be used for encryption
+  account that the Amazon S3 Find and Forget Solution is deployed in and
+  contains encrypted data, a CMK must be used for encryption
```

Comment:
```suggestion
  account that the Amazon S3 Find and Forget Solution is deployed in,
  only SSE-KMS with a customer master key (CMK) may be used for encryption
```

My RFC2119 senses were triggered: it's a bit confusing to use "must" when encryption isn't mandatory.